app/src/main/java/com/f0x1d/logfox/extensions/FABExtensions.kt

```diff
@@ -0,0 +1,13 @@
+package com.f0x1d.logfox.extensions
+
+import androidx.appcompat.widget.TooltipCompat
+import com.google.android.material.floatingactionbutton.FloatingActionButton
+
+fun FloatingActionButton.setDescription(resId: Int) {
```

Comment:
Add `@StringRes` annotation to argument.

Use `=` instead of '{ ... }` here. e. g.
```
fun FloatingActionButton.setDescription(@StringRes resId: Int) = setDescription(context.getString(resId))
```